plugins/workspace-search/src/workspace_search.ts

```diff
@@ -344,7 +349,10 @@ export class WorkspaceSearch implements Blockly.IPositionable {
    */
   private onInput() {
     if (this.searchOnInput) {
-      const inputValue = this.inputElement.value.trim();
+      let inputValue = '';
+      if (this.inputElement) {
+        inputValue = this.inputElement.value.trim();
+      }
```

Comment:
Can you cahnge this to `if (this.searchOnInput && this.inputElement)`? Not sure if that works but I think it'd be more readable.